automation/jenkins_build.sh

```diff
@@ -115,7 +115,7 @@ deploy_build () {
 	if [ -d "${WORKSPACE}/layers/meta-balena/tests" ]
 	then
 		# copy all leviathan/testbot tests from meta-balena to the deploy dir
-		cp -av "${WORKSPACE}/layers/meta-balena/tests" "$_deploy_dir/tests"
+		tar -czvf "$_deploy_dir/tests.tar.gz" "${WORKSPACE}/layers/meta-balena/tests"
```

Comment:
- We need to change the comment and 
- We need to copy the tests.tar.gz to the `$_deploy_dir` as well?

Reply:
The file path for the output tarball is already in the deploy dir (`"$_deploy_dir/tests.tar.gz"`)